lib/seccomp/unittest/Makefile

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+include $(CPP_SRC_HOME)/mk/defines.mk
+
+TARGET=ml_test$(EXE_EXT)
+
+USE_BOOST=1
+USE_BOOST_REGEX_LIBS=1
+
+LIBS:=$(LIB_ML_SECCOMP)
+
```

Comment:
For consistency the linker flags should be added here too, i.e. `LDFLAGS:=$(ML_SECCOMP_LDFLAGS)`